modules/apps/feature-flag/feature-flag-test/src/testFunctional/tests/FeatureFlag.testcase

```diff
@@ -69,7 +69,7 @@ definition {
 
 		task ("And a feature flag is disabled") {
 			if (IsElementPresent(locator1 = "FeatureFlag#TOGGLE_ROW", title_row = "Terms and Conditions pages", toggle_state = "Enabled")) {
-				Check.checkToggleSwitch(
+				ClickNoError(
```

Comment:
Why is this change needed?

Reply:
I think that after I changed the `xpath`, some `Check.checkToggleSwitch` or `Uncheck` stopped working, so I needed to replace it.